tests/integration/suite/test_global_roles.py

```diff
@@ -0,0 +1,24 @@
+def test_builtin_default_can_be_edited(admin_mc, remove_resource):
```

Comment:
This test needs to undo anything it does, a fixture might be appropriate to ensure this happens